pkg/api/validation/schema.go

```diff
@@ -66,6 +67,48 @@ func NewSwaggerSchemaFromBytes(data []byte) (Schema, error) {
 	return schema, nil
 }
 
+func (s *SwaggerSchema) validateList(obj map[string]interface{}) error {
+	items, exists := obj["items"]
+	if !exists {
+		return fmt.Errorf("no items field in %#v", obj)
+	}
+	if _, ok := items.([]interface{}); !ok {
+		return fmt.Errorf("items isn't a slice")
+	}
+	for _, item := range items.([]interface{}) {
```

Comment:
In thinking about this, I'd really like to have this loop check every `item` even if an early one fails.  You can return a `NewAggregate` after the for loop. That way I can get all my failures at once instead of stumbling through one at a time.